config/routechangerconfig.go

```diff
@@ -6,17 +6,27 @@ import (
 	"strings"
 )
 
-type routeChangerConfig struct {
+type routeChangerConfigItem struct {
 	Reg  *regexp.Regexp
 	Repl string
 	Sep  string
 }
 
+type routeChangerConfig []routeChangerConfigItem
+
 func (rcc routeChangerConfig) String() string {
-	if rcc.Reg == nil {
-		return ""
+	result := ""
+	for _, rcci := range rcc {
+		if rcci.Reg != nil {
+			result += fmt.Sprintf("%s%s%s%s%s", rcci.Sep, rcci.Reg,
+				rcci.Sep, rcci.Repl, rcci.Sep)
+		}
+		result += "\n"
+	}
+	if len(result) > 0 {
+		result = result[:len(result)-1] // to remove the last \n
```

Comment:
This is usually done
```go
for i, x := range xx {
  ...
  if i > 0 {
    result += separator
  }
}
```
or you may collect individual value into a slice and then https://pkg.go.dev/strings#Join